### PR TITLE
rm: improve information on lmp customer factory update

### PR DESCRIPTION
**Pull request by vanmaegima**
## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

File: source/reference-manual/linux/linux-update.rst [hunk 1]
```diff
@@ -3,29 +3,39 @@
...
-  git reset --hard HEAD~1
-  git push -f
+If no merge conflicts are found, this script merges your changes and pushes the updated manifest to your Factory, triggering a new platform build. Once published, the update is deployed to your devices following this tag.
+
+.. tip::
+   If something goes wrong, don’t fret! This is why we use version control!
+
+   .. prompt:: bash
+
+      git reset --hard HEAD~1
```
Comment (angolini) [inline source/reference-manual/linux/linux-update.rst:39]: I prefer we recommend a git revert instead here. Would it be possible?

**Review by angolini — APPROVED** [review]
LGTM

File: source/reference-manual/linux/linux-update.rst [hunk 2]
```diff
@@ -3,29 +3,39 @@
 Updating the Linux microPlatform Core
 =====================================
 
-Your factory platform manifest has been separated to make consuming core
-platform updates easier. At Foundries.io we `release`_ Linux microPlatform
-updates early and often in an effort to get the latest security fixes out to
-users.
+FoundriesFactory platform manifest is tailored to make consuming core platform updates easily.
```
Comment (kprosise) [inline source/reference-manual/linux/linux-update.rst:6]:
```suggestion
The FoundriesFactory® platform manifest is tailored to make consuming core platform updates easily.
```

File: source/reference-manual/linux/linux-update.rst [hunk 3]
```diff
@@ -3,29 +3,39 @@
...
+At Foundries.io, we often `release`_ Linux microPlatform (LmP) updates as an effort to get the latest features and security fixes out to users.
```
Comment (kprosise) [inline source/reference-manual/linux/linux-update.rst:7]:
```suggestion
At Foundries.io™, we often `release`_ Linux® microPlatform (LmP) updates as an effort to get the latest features and security fixes out to users.
```

File: source/reference-manual/linux/linux-update.rst [hunk 4]
```diff
@@ -3,29 +3,39 @@
...
-When the new manifest files have been successfully pushed, a new platform build will be triggered, and once published the update can be deployed.
+   git clone https://source.foundries.io/factories/<myfactory>/lmp-manifest.git
+   git clone https://github.com/foundriesio/lmp-tools
+   cd lmp-manifest/
+   git checkout <branch to update>
+   ../lmp-tools/scripts/update-factory-manifest
 
-If something goes wrong, don’t fret! This is why we use version control!::
+.. tip::
+   We recommend testing the update in a separate branch before merging to your active branches. After tested, the changes can be merged to your development branches.
```
Comment (kprosise) [inline source/reference-manual/linux/linux-update.rst:30]:
```suggestion
   We recommend testing the update in a separate branch before merging to your active branches. After testing, the changes can be merged to your development branches.
```

**Review by kprosise — APPROVED** [review]
Provided suggestions, otherwise it LGTM

**Comment by doanac** [comment]
Docs for b33259c65c1c0a3f81f15cf85e95899af44c6885 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2058/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for c9ce2f66b6b89c0585338d42e55d6b0d98058c6e are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2059/docs/artifacts/html/index.html

**Pull request merged**